scss/datetime/_layout.scss

```diff
@@ -118,4 +139,95 @@
 
     }
 
+    // Infinite timepicker
+    // Header
+    .k-time-header { // Same as the calendar header
```

Comment:
After further discussion with @ggkrustev , we decided to use <button> for the Now action in the timepicker, which differs from the Calendar component (where <span> is used). I will modify the styling in the timepicker header